lib/src/test/java/de/edux/core/math/matrix/classic/MatrixArithmeticTest.java

```diff
@@ -0,0 +1,64 @@
+package de.edux.core.math.matrix.classic;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import de.edux.core.math.IMatrixArithmetic;
+import org.junit.jupiter.api.Test;
+
+class MatrixArithmeticTest {
+
+  @Test
+  public void testLargeMatrixMultiplication() {
```

Comment:
How fast is this? Will GitHub JUnit Test Build Step finish? Maybe extract size into a separate variable and set it to 200.